### PR TITLE
General trinary types cleanup, and some more

### PR DESCRIPTION
**Pull request by Laboredih123**
[ATMOSPHERICS][CLEANLINESS][INTERNAL][BUG]
## About the PR 
Creates the trinary subtype, moves trinary things to the subtype, cleans up and adds to their files, and perhaps fixes some icons.

## Why's this needed? 
I prefer my clean and performant code and correct icons.

### CODE REVIEW

**Comment by Laboredih123** [comment]
i forgot the last thing i was gonna do so im just gonna open

File: code/modules/atmospherics/machinery/trinary/manifold_valve.dm [hunk 1]
```diff
@@ -0,0 +1,140 @@
+
+/obj/machinery/atmospherics/trinary/manifold_valve
+	icon = 'icons/obj/atmospherics/manifold_valve.dmi'
+	icon_state = "manifold_valve0"
+
+	name = "manifold valve"
+	desc = "A pipe valve"
+
+	var/divert = FALSE
```
Comment (TheTyrant25) [inline code/modules/atmospherics/machinery/trinary/manifold_valve.dm:9]: might be worth doc commenting what divert does

File: code/modules/atmospherics/machinery/trinary/manifold_valve.dm [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+	var/id = null
+
+/obj/machinery/atmospherics/trinary/manifold_valve/New()
+	..()
+	qdel(src.air1)
+	src.air1 = null
+	qdel(src.air2)
+	src.air2 = null
+	qdel(src.air3)
+	src.air3 = null
```
Comment (TheTyrant25) [inline code/modules/atmospherics/machinery/trinary/manifold_valve.dm:21]: not sure why this nullifying is necessary?

File: code/modules/atmospherics/machinery/trinary/mixer.dm [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+
+/obj/machinery/atmospherics/trinary/mixer
+	name = "Gas mixer"
+	icon = 'icons/obj/atmospherics/mixer.dmi'
+	icon_state = "intact_off"
+	layer = PIPE_MACHINE_LAYER
+	plane = PLANE_NOSHADOW_BELOW
+
+	var/id_tag
+	var/master_id
```
Comment (TheTyrant25) [inline code/modules/atmospherics/machinery/trinary/mixer.dm:15]: what are these id tags for btw

File: code/modules/atmospherics/machinery/trinary/mixer.dm [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+
+	var/target_pressure = ONE_ATMOSPHERE
+	var/node1_ratio = 0.5
+	var/node2_ratio = 0.5
+
+	var/frequency
+
+/obj/machinery/atmospherics/trinary/mixer/New()
+		..()
+		air3.volume = 300
```
Comment (TheTyrant25) [inline code/modules/atmospherics/machinery/trinary/mixer.dm:26]: why is this not just the default value declared higher up? Does something in a parent override it?

**Review by TheTyrant25 — COMMENTED** [review]
Didn't read retrofilter or trinary base, but most of it looks okay. I do have some questions on how it all works, but at a surface level glance over by somebody not familiar with the code, all I can really say is "use src.variablename more please" and "doc comment more vars and procs".

File: code/modules/atmospherics/machinery/trinary/manifold_valve.dm (hunk 2 above)
Comment (Laboredih123) [reply code/modules/atmospherics/machinery/trinary/manifold_valve.dm:21]: we don't use these gasmixes, so i disposed of them to save memory. It's explicitly set to null to clear our reference to it.

File: code/modules/atmospherics/machinery/trinary/mixer.dm (hunk 3 above)
Comment (Laboredih123) [reply code/modules/atmospherics/machinery/trinary/mixer.dm:15]: Asking stuff and modifying what the mixer mixes.

File: code/modules/atmospherics/machinery/trinary/mixer.dm (hunk 4 above)
Comment (Laboredih123) [reply code/modules/atmospherics/machinery/trinary/mixer.dm:26]: The parent usually does 200L, the mixer uses 300L

File: code/modules/atmospherics/machinery/trinary/manifold_valve.dm (hunk 1 above)
Comment (Laboredih123) [reply code/modules/atmospherics/machinery/trinary/manifold_valve.dm:9]: ight

File: code/modules/atmospherics/machinery/trinary/manifold_valve.dm (hunk 2 above)
Comment (TheTyrant25) [reply code/modules/atmospherics/machinery/trinary/manifold_valve.dm:21]: fair enough

File: code/modules/atmospherics/machinery/trinary/mixer.dm (hunk 4 above)
Comment (TheTyrant25) [reply code/modules/atmospherics/machinery/trinary/mixer.dm:26]: does `air3.volume = 300` not work in the `/obj/machinery/atmospherics/trinary/mixer` part then

Comment (Laboredih123) [reply code/modules/atmospherics/machinery/trinary/mixer.dm:26]: gotta do it in New, cause otherwise the gas doesn't exist yet.

Comment (TheTyrant25) [reply code/modules/atmospherics/machinery/trinary/mixer.dm:29]: i see

**Pull request merged**